### ISSUE TITLE
China having problem accessing release (bazel.exe)

### ISSUE THREAD

**Issue by veyvin** [bazelbuild/bazel#2922]

**Comment by damienmg** [comment]
I don't know there is anything we can do here but I know TensorFlow had similar issue. /cc @jart @steren 

(To be clear: this is for accessing release stored on github).

**Comment by steren** [comment]
GitHub should not be the unique source of truth for release binaries. Let's also serve them from our Google Cloud Storage bucket, which we already use with apt.

**Comment by damienmg** [comment]
This was actually already pushed for the next release. GCS isn't served from china unfortunately :(

**Comment by steren** [comment]
Let's serve our static files under the bazel.build domain name. I suppose this should unblock the situation.

**Comment by damienmg** [comment]
SGTM

**Comment by damienmg** [comment]
let's do that for our mirroring too. That will help tensorflow too :)

**Comment by veyvin** [comment]
yeap, some files in github was store in aws s3,these files in china can't download , so can I have another way to download bazel.exe

**Comment by jart** [comment]
I remember talking to the Go team and they said that they went through a lot of trouble to get the Go release files to download in China. It might be worth syncing up with them to ask what they did.

**Comment by damienmg** [comment]
FTR I added the GCE config so we are serving it but we are waiting on dns update.

**Comment by damienmg** [comment]
DNS change is in, now we need to wait for DNS to propagate.

**Comment by damienmg** [comment]
FYI it is now live, https://releases.bazel.build/0.5/rc3/ will show you the rc for 0.5. Starting 0.5 we will also put the final release on that site. There is still some work that can be done to make it easier to navigate but that's pretty low priority. I'll keep that bug open until I test that from china when I will be there.

@jart: also the mirror is now served from https://mirror.bazel.build

**Comment by jart** [comment]
@damienmg I'm happy to refactor everything to use the new hostname, so long as the old one continues to work forever. OOC does mirror.bazel.build work in China?

**Comment by damienmg** [comment]
IIUC it should work from China (same way of serving than our website). I'll
be in China in a month so I'll check first hand :)

The old URL still exists no worries.

**Comment by damienmg** [comment]
Double check, it is working (in Shanghai now). Closing that issue

**Comment by cooperpellaton** [comment]
@damienmg Would it be possible to publicly share the links of the releases on mirror.bazel.build somewhere? Maybe even potentially on the releases page? 

I'm in Shanghai now trying to take guesses at the working links for releases.bazel.build but can't quite seem to guess correctly. I'd greatly appreciate some transparency in the use of the official mirror.

**Comment by damienmg** [comment]
https://releases.bazel.build/0.5.2/release/index.html is the link to the
latest release.

Yes, we plan to make those links the official one and share them everywhere
(in release mail and from bazel.build), they are faster and have better
availability than github.
